build.gradle.kts

```diff
@@ -17,7 +17,9 @@ dependencies {
 	implementation("com.github.kittinunf.result:result:3.1.0")
 	implementation("com.github.kittinunf.result:result-coroutines:3.1.0")
 	implementation("com.beust:klaxon:5.4")
-	implementation("com.amazonaws:aws-java-sdk-s3:1.11.873")
+	implementation("com.amazonaws:aws-java-sdk-s3:1.11.874")
+
+	testImplementation("org.mockito:mockito-inline:3.5.13")
```

Comment:
This removes the need to have something specific in the test resource folder.  It enables mock of final classes (which most are in Kotlin).